views/js/chatControl.js

```diff
@@ -2,8 +2,44 @@ var chatActiveRoom = "general";
 var chatRefreshTimeMs = 30000;
 
 var refreshChat = function () {
-  document.getElementById("chat_messages").innerHTML = fetchChatHTML();
-  setChatScrollToBottom();
+  fetch('./chat/' + chatActiveRoom)
+    .then(function (response) {
```

Comment:
Prefer `(response) => {` syntax?

Reply:
Done https://github.com/knative-portability/Kubercade/pull/22/commits/0734a1dfacdc5c3ed50b8ba71cdc56aa1bc36027

Reply:
This is still a `function() {}`

Reply:
Oops, fixed everywhere except where you pointed out... done https://github.com/knative-portability/Kubercade/commit/7488a5b34816ae2f301f453a9bc4cc02341f5695